packages/fake-driver/package.json

```diff
@@ -21,24 +21,25 @@
   },
   "license": "Apache-2.0",
   "author": "https://github.com/appium",
-  "main": "./build/index.js",
+  "main": "fake-driver.js",
   "bin": {
```

Comment:
i'm surprised we have a bin dir for fake driver, i don't think we need it?

Reply:
idk.  it was there before I got here 😄